### PR TITLE
Cedict

### PR DESCRIPTION
**Pull request by kirlat**
Updated to add retrieval of Chinese data. The way how Chinese Lexical Queries is handled now is temporary: it is planned to be replaced with a more consistent approach according to our future architectural changes. It will be implemented in the next phase.

### CODE REVIEW

File: src/vue/components/info.vue [hunk 1]
```diff
@@ -40,38 +41,42 @@
...
     swapPosition: SwapPosition,
     tapGestureIcon: TapGestureIcon
   },
   mixins: [DependencyCheck],
+
+  data: () => {
+    return {
+      cedictResponse: ''
+    }
+  },
```
Comment (balmas) [inline src/vue/components/info.vue]: what is the reason for this change ?

File: src/lib/controllers/ui-controller.js [hunk 2]
```diff
@@ -1934,11 +1974,12 @@ export default class UIController {
           customEv = this.options.textQueryTriggerDesktop
       }
     }
+    const lexisModule = this.getModule('lexis')
     if (ev) {
-      this.evc.registerListener(listenerName, selector, this.getSelectedText.bind(this), ev)
+      this.evc.registerListener(listenerName, selector, this.api.lexis.getSelectedText.bind(lexisModule), ev)
```
Comment (balmas) [inline src/lib/controllers/ui-controller.js]: so, do I understand correctly that right now, the getSelectedText handler in the lexis module is just a reference back to the getSelectedText method in UIController? And the intent is for that code eventually to move into the lexis module itself?

File: src/vue/components/info.vue (hunk 1 above)
Comment (kirlat) [reply src/vue/components/info.vue]: This is a leftover from a test code that was showing results of a CEDICT request in an `info` component. This is not needed now and I will remove it. Thanks for a good catch!

File: src/lib/controllers/ui-controller.js (hunk 2 above)
Comment (kirlat) [reply src/lib/controllers/ui-controller.js]: Yes, that's exactly it. All lexical requests are supposed to go through the Lexis data module and corresponding code from a UI controller will be moved there. So this is a temporary workaround until then.
